### DISCUSSION TITLE
Differences in information content (fields) in different pull export formats

### DISCUSSION THREAD

**Discussion by rkrug** [retorquere/zotero-better-bibtex#2590] [General]
Better BibTex has four formats which I think are suitable for [pull export](https://retorque.re/zotero-better-bibtex/exporting/pull/) (the other two, `jzon` and `translatorID` seem to specialised for the moment) for my use case:

- biblatex for BibLaTeX
- bibtex for BibTeX
- json or csljson for CSL-JSON
- yaml, yml or cslyaml for CSL-JSON in YAML format

What are the differences in the default usage of these in regards to fields exported?

**Comment by retorquere** [comment]
I don't understand the question, sorry. They export the same as when you export manually, but I don't know what is meant by default usage, or which fields you refer to (zotero fields? Bibtex/csl fields?)
